cts/cts-attrd.in

```diff
@@ -183,6 +183,33 @@ class AttributeTests(Tests):
                                   "name=\"DDD\" value=\"555\"")
         test.add_log_pattern("Processed 1 private change for DDD, id=n/a, set=foo")
 
+    def build_multiple_query_tests(self):
+        """ Add tests that set and query an attribute across multiple nodes """
+
+        # NOTE:  These tests make use of the fact that nothing in attrd actually
+        # cares about whether a node exists when you set or query an attribute.
+        # It just keeps creating new hash tables for each node you ask it about.
+
+        test = self.new_test("multi_query_1",
+                             "Query an attribute set across multiple nodes")
+        test.add_cmd("attrd_updater", "--name AAA -U 111 --node cluster1 --output-as=xml")
+        test.add_cmd("attrd_updater", "--name AAA -U 111 --node cluster2 --output-as=xml")
```

Comment:
use different values to verify that it's only setting the attribute for the one node

Reply:
Good idea, fixed.